### ISSUE TITLE
RotaryCraft Bee Species Incompatibility [Forestry 4.0]

### ISSUE THREAD

**Issue by ghost** [ForestryMC/ForestryMC#849]
Using 4.0.0 version and latest RotaryCraft.
http://pastebin.com/YbTHBsuu

**Comment by bakaxyf** [comment]
You should report to [Reika_Mods_Issues](https://github.com/ReikaKalseki/Reika_Mods_Issues/issues).

**Comment by mezz** [comment]
Thanks for the report!
Reika's DragonAPI is not compatible with Forestry 4.0 beta yet.

**Comment by mezz** [comment]
Leaving this open so it's visible for people coming to report bugs.

**Comment by bakaxyf** [comment]
https://github.com/ReikaKalseki/Reika_Mods_Issues/issues/229 
https://github.com/ReikaKalseki/Reika_Mods_Issues/issues/285 
https://github.com/ReikaKalseki/Reika_Mods_Issues/issues/287 
https://github.com/ReikaKalseki/Reika_Mods_Issues/issues/289

**Comment by mezz** [comment]
DragonAPI has dependencies on code internal to Forestry. 
It will probably take a while for Reika to sort it out.

**Comment by AlexPlus117** [comment]
Are there any news about fixing this crash?

**Comment by mezz** [comment]
Unfortunately not really, no. If you want to stay up to date you can take a look here: https://github.com/ReikaKalseki/Reika_Mods_Issues/issues/229

TLDR:
Reika is aware of the issue, but DragonAPI has a complicated maintenance schedule because it is integrated with so many mods, and other people depend on it for their projects.

**Comment by AlexPlus117** [comment]
Thanks, I'm staying tuned :D

**Comment by mezz** [comment]
DragonAPI was updated a while ago and I forgot to close this issue, sorry.
